src/liteclient.cc

```diff
@@ -78,13 +77,14 @@ LiteClient::LiteClient(Config& config_in, const AppEngine::Ptr& app_engine, bool
   headers.emplace_back("x-ats-tags: " + boost::algorithm::join(tags, ","));
 
   http_client = std::make_shared<HttpClient>(&headers);
-  uptane_fetcher_ = std::make_shared<Uptane::Fetcher>(config, http_client);
-  report_queue = std_::make_unique<ReportQueue>(config, http_client, storage);
 
   key_manager_ = std_::make_unique<KeyManager>(storage, config.keymanagerConfig());
   key_manager_->loadKeys();
   key_manager_->copyCertsToCurl(*http_client);
 
+  uptane_fetcher_ = std::make_shared<Uptane::Fetcher>(config, http_client);
+  report_queue = std_::make_unique<ReportQueue>(config, http_client, storage);
```

Comment:
this is that long standing bug where we saw weird ssl stuff at start up, right?

Reply:
I hope so :)